decentralizedGoToServerScript.js

```diff
@@ -8,14 +8,16 @@
 //  See the accompanying file LICENSE or https://opensource.org/licenses/ISC
 
 (function () {
+    var ipJsonUrl = Script.resolvePath("ip.json");
+    var webSocketUrl = ipJsonUrl.split("/")[2].split(":")[0];
+    var webSocketPort = ""; // Leave empty if no specific port.
+    var pathToWS = webSocketPort + "/interim/d-goto/ws";
```

Comment:
What's the point of adding the `webSocketPort`?

Reply:
To offer a clear path for anyone modifying the script for their own use or deployment. It's not really necessary, just a quality of life thing I saw I had stashed in the repo from months ago.